openrtb_ext/request_wrapper.go

```diff
@@ -1298,6 +1300,13 @@ func (e *ImpExt) unmarshal(extJson json.RawMessage) error {
 		}
 	}
 
+	tidJson, hasTid := e.ext["tid"]
+	if hasTid {
+		if err := json.Unmarshal(tidJson, &e.tid); err != nil {
```

Comment:
Could you add a test where `if hasTid` is true? I noticed that the code block where it is true isn't covered by tests. I'm not looking for the unmarshal return err line (line 1306) to be covered, but line 1305 should be covered, just to show that `hasTid` being true is being tested.

Reply:
Yep, just added test coverage around this block